packages/stable/src/api/vision/types.gen.ts

```diff
@@ -56,16 +66,46 @@ export interface AnnotationsCogniteAnnotationTypesImagesAssetLink {
      * @max 1
      */
     confidence?: number;
+    /** The region of the object representing the asset */
+    objectRegion?: AnnotationsCogniteAnnotationTypesPrimitivesGeometry2DGeometry;
     /** The extracted text */
     text: string;
     /** The location of the text mentioning the asset */
     textRegion: AnnotationsBoundingBox;
 }
 /**
+* A geometry represented by exactly *one of* ` bounding_box`, `polygon` and
+`polyline` which, respectively, represents a BoundingBox, Polygon and
+PolyLine.
+*/
+export interface AnnotationsCogniteAnnotationTypesPrimitivesGeometry2DGeometry {
+    /** A plain rectangle */
+    boundingBox?: AnnotationsBoundingBox;
+    /**
+     * A _closed_ polygon represented by _n_ vertices. In other words, we assume
+     * that the first and last vertex are connected.
+     */
+    polygon?: AnnotationsPolygon;
+    /** A polygonal chain consisting of _n_ vertices */
+    polyline?: AnnotationsPolyLine;
+}
+/**
+ * The numerical value of something
+ */
+export interface AnnotationsNumerical {
+    /** The description of a primitive */
+    description?: string;
+    type: "numerical";
+    /** The numerical value */
+    value: number | number;
```

Comment:
`number | number` -- interesting 😅 . Since TSC is not complaining, then I guess this is fine

Reply:
Maybe you want to update your openapi spec:
![image](https://github.com/cognitedata/cognite-sdk-js/assets/7851860/0ace45c8-b6d6-4c41-b5f2-5174ffa2019b)